spring-integration-core/src/test/java/org/springframework/integration/handler/LoggingHandlerTests.java

```diff
@@ -126,6 +129,23 @@ public void testChangeLevel() {
 		verify(log, times(1)).warn(Mockito.anyString());
 	}
 
+	@Test(expected = MessageHandlingException.class)
```

Comment:
Maybe, but not in all cases...
Especially when I would like to verify several calls in a single test-case.